examples/sampling/demo_diffpir.py

```diff
@@ -25,7 +25,6 @@
 # Generate an inverse problem
 # ---------------------------
 #
-
```

Comment:
why is this?

Reply:
Because the note was not showing in the docs, see https://deepinv.github.io/deepinv/auto_examples/sampling/demo_diffpir.html#generate-an-inverse-problem